shopfloor_mobile/static/wms/src/components/scenario_picking_detail/picking_select.js

```diff
@@ -6,7 +6,7 @@
 /* eslint-disable strict */
 /* eslint-disable no-implicit-globals */
 import {PickingDetailSelectMixin} from "./mixins.js";
```

Comment:
Any reason this one did nog get changed to an absolute path, like the rest ?

Reply:
yes, one file is local and one not. `mixins.js` here is the one from `shopfloor_mobile`